utils/spacewalk-common-channels.ini

```diff
@@ -1071,3 +1143,42 @@ repo_type = deb
 checksum = sha256
 base_channels = debian-9-pool-amd64
 repo_url = http://ftp.debian.org/debian/dists/stretch-updates/main/binary-amd64/
+
+[debian-9-amd64-main-security]
+label    = debian-9-amd64-main-security
+name     = Debian 9 (stretch) AMD64 Main Security
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-9-pool-amd64
+repo_url = http://security-cdn.debian.org/debian-security/dists/stretch/updates/main/binary-amd64/
+
+[debian-10-pool-amd64]
+label    = debian-10-pool-amd64
+checksum = sha256
+archs    = amd64-deb
+repo_type = deb
+name     = Debian 10 (buster) pool for amd64
+gpgkey_url =
+gpgkey_id =
+gpgkey_fingerprint =
+repo_url = http://ftp.debian.org/debian/dists/buster/main/binary-amd64/
```

Comment:
Please change this to http://ftp.debian.org/debian/dists/buster/main/binary-amd64/ so that it uses a CDN mirror